prow/cmd/deck/static/script.js

```diff
@@ -448,7 +448,10 @@ function redraw(fz) {
         if (!jobCountMap.has(build.state)) {
           jobCountMap.set(build.state, 0);
         }
-        totalJob += 1;
+        totalJob ++;
+        if (totalJob > 499) {
+            continue;
+        }
         jobCountMap.set(build.state, jobCountMap.get(build.state) + 1);
```

Comment:
Move this line above the `totalJob` conditional so that we get stats for all job, not just the truncated ones. (see my later comment).